vertx-pojo-mapper-common/src/main/java/de/braintags/vertx/jomnigate/init/ObserverMapperSettings.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * #%L
+ * vertx-pojo-mapper-common
+ * %%
+ * Copyright (C) 2017 Braintags GmbH
+ * %%
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * #L%
+ */
+package de.braintags.vertx.jomnigate.init;
+
+import java.lang.annotation.Annotation;
+
+import de.braintags.vertx.jomnigate.exception.MappingException;
+import de.braintags.vertx.jomnigate.mapping.IMapper;
+
+/**
+ * The part of {@link ObserverSettings} which defines, on which {@link IMapper} the ObserverSettings shall be executed.
+ * The property classDefinition and annotation contain the definition. Examples for a class definition are:
+ * <UL>
+ * <LI>my.mapper.class<br/>
+ * the parent ObserverSettings will be applied for a mapper with the class my.mapper.class
+ * <LI>instanceof my.mapper.class<br/>
+ * the parent ObserverSettings will be applied for all classes, which are instanceof my.mapper.class
+ * </UL>
+ * 
+ * @author Michael Remme
+ * 
+ */
+public class ObserverMapperSettings {
+  private static final String INSTANCEOF = "instanceof";
+  private String classDefinition;
+  private boolean instOf = false;
+  private Class<?> mapperClass;
+  private Class<? extends Annotation> annotation;
+
+  @SuppressWarnings("unused")
+  private ObserverMapperSettings() {
+    // only usable for serialization
+  }
+
+  /**
+   * Constructor for a new instance. The class definition can be something like "my.mapper.class" or "instanceof
+   * my.mapper.class"
+   * 
+   * @param classDefinition
+   */
+  public ObserverMapperSettings(String classDefinition) {
+    setClassDefinition(classDefinition);
+  }
+
+  /**
+   * Constructor for a new instance, which shall be executed on mapper classes, which are annotated with the given
+   * annotation
+   * 
+   * @param annotation
+   */
+  public ObserverMapperSettings(Class<? extends Annotation> annotation) {
+    setAnnotation(annotation);
+  }
+
+  private void init() {
+    if (classDefinition != null) {
+      String clsName = classDefinition;
+      int index = clsName.indexOf(INSTANCEOF);
+      if (index >= 0) {
+        instOf = true;
+        clsName = clsName.substring(index + INSTANCEOF.length()).trim();
+      }
```

Comment:
Serialization of ObserverMapperSettings is done per jackson already. I thought about to handle instanceof as further field, but decided to let this information inside the context, where it is belonging to, cause it can happen that there are arriving further arguments like "package" for instance. And in this coming case i prefer to write and read something like 

`"classDefinition" : "instanceof myClass" `

than
`
"classDefinition" : "myClass",
"instanceof" : "true",
"package" : "false"
`